hicommonwealth/commonwealth
# `[Launchpad Discovery]`: Launch token from communities explore page

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/8802

## Description of Changes
Added the ability to launch a token from the communities explore page in a quick/streamlined flow.


https://github.com/user-attachments/assets/2302e9d6-f61c-4f30-93f6-0b9cbdfb1fff



## "How We Fixed It"
N/A

## Test Plan
1. Enable `FLAG_TOKENIZED_COMMUNITY`
2. Visit `/communities`
3. Verify you see a token launch card, click on the "Launch token button" and verify a drawer opens with the token information form
4. Fill out the token information form and click on "Launch Token" -- if you attempt to close the browser tab at this moment, verify you see a browser notification asking for confirmation, otherwise if you try to close the drawer at this moment, verify you see a confirmation dialog.
5. You'd be prompted to sign a signature request. 
     a. If you sign the request, verify your token is launched and a success screen shows up with a redirect button to the complementary community created.
     b. If you deny the signature request, submit the form again, and then sign the request, verify that once the token is created, there are no duplicate communities (communities with matching name: ex: `token-(tkn)-community` and `token-(tkn)-community1` exists). We optimize the submit process to not create multiple communities referencing the same token.
6. Verify that after disabling `FLAG_TOKENIZED_COMMUNITY`, you don't see token launch card on communities page
7. Verify mobile responsiveness for the components.

## Deployment Plan
N/A

## Other Considerations
N/A
